pycm/pycm_curve.py

```diff
@@ -96,9 +98,55 @@ def area(self, method="trapezoidal"):
                 raise pycmCurveError(AREA_METHOD_ERROR)
         return self.auc
 
-    def plot(self):
-        """Plot method."""
-        pass
+    def plot(
+            self,
+            classes=None,
+            area=False,
+            area_method="trapezoidal",
+            colors=None,
+            markers=None,
+            linewidth=1):
+        """
+        Plot the given curve.
+
+        :param classes: ordered labels of classes
+        :type classes: list
+        :param area: area flag
+        :type area: bool
+        :param area_method: numerical integral technique (trapezoidal or midpoint)
+        :type area_method: str
+        :param colors: color for each class in plot
+        :type colors: list
+        :param markers: plot marker
+        :type markers: list
+        :param linewidth: plot line width
+        :type linewidth: float
+        :return: plot axes
+        """
+        fig, ax, classes = __plot_validation__(
+            self, classes, area, area_method, colors, markers)
+        ax.set_xlabel(self.plot_x_axis)
+        ax.set_ylabel(self.plot_y_axis)
+        fig.suptitle(self.title)
+        for c_index, c in enumerate(classes):
+            label = "class {}".format(c)
```

Comment:
Here, I think it would be better to remove that `class`. Consider this case :
```
classes = ["Class1", " Class2"]
```

Output: ```class Class1```